sympy/simplify/epathtools.py

```diff
@@ -52,7 +52,7 @@ def __new__(cls, path):
             index = 0
 
             for c in selector:
-                if c.isalnum() or c == '_' or c == '|' or c == '?':
+                if c.isalnum() or c in ('_', '|', '?'):
```

Comment:
Checks like this could also be `c in '_|?'`. No preference though, as one could consider it to be less readable.

Reply:
Say some instance of it, but, again, just went for a minimum change.

Reply:
That's not equivalent:
```python
>>> 'ab' in ('a', 'b')
False
>>> 'ab' in 'ab'
True
```